app/assets/javascripts/discourse/app/components/search-menu/highlighted-search.hbs

```diff
@@ -1 +1 @@
-{{this.content}}
\ No newline at end of file
+<span>{{html-safe @string}}</span>
```

Comment:
Curious, do we need the extra `span`?

Reply:
Fair point, this was carried over from the previous implementation so I didn't second guess it.

Looking at the html structure you can definitely see that it is unnecessary.
<img width="293" alt="Screenshot 2023-06-16 at 3 40 17 PM" src="https://github.com/discourse/discourse/assets/50783505/e11e2be6-bddf-42f7-b09f-41f44c7749e7">

I will remove that.